spyderlib/userconfig.py

```diff
@@ -113,7 +115,7 @@ def _save(self):
         def _write_file(fname):
             if PY2:
                 # Python 2
-                with open(fname, 'w') as configfile:
+                with codecs.open(fname, 'w', encoding='utf-8') as configfile:

```

Comment:
According to this [StackOverflow answer](http://stackoverflow.com/a/10975371/438386), it's better to use `io.open`, instead of `codecs.open` because we don't support Python 2.5 anymore.

Could you change this and the call below to `io.open` please?